storage/engine/mvcc.go

```diff
@@ -94,6 +94,7 @@ func (k MVCCKey) IsValue() bool {
 
 // EncodedSize returns the size of the MVCCKey when encoded.
 func (k MVCCKey) EncodedSize() int {
+	// TODO(peter): This should now be "+ 1". Fix me.
```

Comment:
Or soon thereafter. I'll go ahead and fix it here. It just requires adjustment to a few tests.